pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -56,6 +56,14 @@ const (
 
 	// LocalhostAddress for local binding
 	LocalhostAddress = "127.0.0.1"
+
+	// EnvoyLogFormat format for envoy access logs
+	EnvoyLogFormat = "[%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%" +
```

Comment:
I think you can use backticks here to make this a raw string literal. Would need less escape chars.